CollapsibleExample/MHCollapsibleSection.h

```diff
@@ -0,0 +1,63 @@
+//
+//  MHCollapsibleSection.h
+//  CollapsibleExample
+//
+//  Class that handles a section of data corresponding to sells
+//  Note: section does not mean the same as UITableView section
+//  but rather a combo of header + child data
+//  Created by Lizzy Randall on 5/19/15.
+//  Copyright (c) 2015 Lizzy Randall. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
+#import "MHFilterLabel.h"
+
+@interface MHCollapsibleSection : NSObject
+
+//Setters
+- (instancetype) initWithArray:(NSArray*)filters headerTitle:(NSString*)headerTitle
+                     animation:(UITableViewRowAnimation)animation rowRange:(NSRange)rowRange;
+
+//Check methods
+
+- (BOOL)checkStateOfRowWithIndexRow:(NSUInteger)row;
+
+//returns result of if row number is in range for section
+- (BOOL)rowNumInRange:(NSUInteger)num;
+
+//returns label at a specified row, note: this is offset since the labels themselves
```

Comment:
Great naming on all these methods. I also like your commenting. A helpful tool that will make your commenting quicker and format your comments according to apple's standards is VVDocumenter. If you install alcatraz (an xcode plugin manager, http://alcatraz.io/ ) and search for it I think you will enjoy it. You should also look around alcatraz for plugins that make your life easier.